ipl-auction-react/src/control/outLinedTextField.css

```diff
@@ -0,0 +1,43 @@
+.textfield {
+  position: relative;
+}
+
+label {
+  position: absolute;
+  font-size: 1rem;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  background-color: white;
+  color: gray;
+  padding: 0 0.3rem;
+  margin: 0 0.5rem;
+  transition: 0.1s ease-out;
+  transform-origin: left top;
+  pointer-events: none;
+}
+
+input {
+  font-size: 1rem;
+  outline: none;
+  border: 2px solid gray;
+  border-radius: 5px;
+  padding: 1rem 0.7rem;
+  color: gray;
+  transition: 0.1s ease-out;
+}
+
+input:focus {
+  border-color: #15327d;
+}
+
+input:focus + label {
+  color: #15327d;
+  top: 0;
+  transform: translateY(-50%) scale(0.9);
+}
+
+input:not(:placeholder-shown) + label {
```

Comment:
what it does place-shown

Reply:
placeholder-shown is used to style the placeholder text in the input box. In my CSS class, I'm not showing the placeholder in the input box.